Doc/library/time.rst

```diff
@@ -274,17 +274,24 @@ Functions
    flag is set to ``1`` when DST applies to the given time.
 
 
-.. function:: mktime(t)
+.. function:: mktime(t) -> int
 
    This is the inverse function of :func:`localtime`.  Its argument is the
    :class:`struct_time` or full 9-tuple (since the dst flag is needed; use ``-1``
    as the dst flag if it is unknown) which expresses the time in *local* time, not
-   UTC.  It returns a floating point number, for compatibility with :func:`.time`.
+   UTC.
+
+   Return the time in seconds since the epoch_ as an integer. The specific date
```

Comment:
I'm wondering if this sentence should come before "This is the inverse function of ..."